sphinx_rtd_theme/__init__.py

```diff
@@ -58,6 +58,9 @@ def setup(app):
         # enabled at most once.
         # See: https://www.sphinx-doc.org/en/master/extdev/appapi.html#sphinx.application.Sphinx.setup_extension
         app.setup_extension("sphinxcontrib.jquery")
+        # However, we need to call the extension's callback since setup_extension doesn't do it
+        from sphinxcontrib.jquery import add_js_files as jquery_add_js_files
+        jquery_add_js_files(app, app.config)
```

Comment:
This should probably be called inside `config-initied` since it requires the config to be initialized first.

Reply:
see PR description https://github.com/readthedocs/sphinx_rtd_theme/pull/1448#issue-1632274671